### PR TITLE
chore(build): run acceptance tests in parallel

### PR DESCRIPTION
**Pull request by atabel**
## Parallelized ci build job
- `build`: lint, prettier, circular depenencies, build, ts-check, tests, ssr-tests
- `build-acceptance` x3 (sharded with `jest --shard`): storybook-static, test-acceptance
  - once the 3 build-acceptance jobs have finished, `build-upload-failed-screenshots` job runs

image

## Split the deploy-pull-request job
- `deploy-pull-request` only deploys to vercel
  - Don't build `/playroom-mobile` and `/playroom-desktop` in PR previews, only in production deploys
- new `audit-a11y` job

### CODE REVIEW

**Review by sergivillar — APPROVED** [review]

**Review by brtbrt — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,14 +27,56 @@ jobs:
...
             - run: yarn build
             - run: yarn ts-check
-            - run: yarn storybook-static
             - run: yarn test --ci
             - run: NO_BUILD=true yarn test-ssr --ci
-            - run: yarn test-acceptance --ci
+
+    build-acceptance:
+        runs-on: ubuntu-latest
+        container: telefonica/web-builder:chromium93.0.4577-1.2.4
```
Comment (pladaria) [inline .github/workflows/ci.yml:35]: you could update to use the latest image: `chromium93.0.4577-node18-1.0.0`

**Comment by pladaria** [comment]
nice!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (atabel) [reply .github/workflows/ci.yml:35]: I tried, but had to revert because vercel doesn't support node 18 yet

**Pull request merged**